### PR TITLE
Update README with instructions what to use instead of this action

### LINKED ISSUE #9: Issues to project action fails when creating new issue
# How to reproduce

## Prerequisite

Github repo with following action

```yml
name: Add new issue to Vuorovaikutustiimi project
on:
  issues:
    types:
      - opened
      - reopened

jobs:
  add_issue_to_project:
    name: 'Add issue to project board (new, reopened, labelled)'
    runs-on: ubuntu-latest
    steps:
      - uses: Yleisradio/gh-project-actions/issues-to-project@v1.1.0
        with:
          token: ${{secrets.GH_ACTIONS_TOKEN}}
          project-id: 17
          issue-id: ${{ github.event.issue.node_id }}
          target-column: Inbox
```

## Steps

1. Add new Issue in the repo

## Expectation

Issue is added to github board

## Actual

Action fails with error

```
Run $GITHUB_ACTION_PATH/issues_to_project.sh
- Get project and field data
gh: Field 'projectNext' doesn't exist on type 'Organization'
Variable $number is declared by anonymous query but not used
jq: error (at project_data.json:0): Cannot iterate over null (null)
- Looking for issue I_kwDOA_uAiM5eUjuu in project null
gh: No such type ProjectNext, so it can't be a fragment condition
Variable $endCursor is declared by anonymous query but not used
jq: error (at board_issues.json:0): Cannot iterate over null (null)
- Issue not found in project
- Add Issue to the project board
gh: Field 'addProjectNextItem' doesn't exist on type 'Mutation'
Variable $project_id is declared by anonymous mutation but not used
Variable $content_id is declared by anonymous mutation but not used
- New board item id is {"errors":[{"path":["mutation","addProjectNextItem"],"extensions":{"code":"undefinedField","typeName":"Mutation","fieldName":"addProjectNextItem"},"locations":[{"line":3,"column":9}],"message":"Field 'addProjectNextItem' doesn't exist on type 'Mutation'"},{"path":["mutation"],"extensions":{"code":"variableNotUsed","variableName":"project_id"},"locations":[{"line":2,"column":7}],"message":"Variable $project_id is declared by anonymous mutation but not used"},{"path":["mutation"],"extensions":{"code":"variableNotUsed","variableName":"content_id"},"locations":[{"line":2,"column":7}],"message":"Variable $content_id is declared by anonymous mutation but not used"}]}.
```

## Example

https://github.com/Yleisradio/tehtava-ui/actions/runs/4164423336/jobs/7206039181

### PR DESCRIPTION
**Pull request by rpsu**

### CODE REVIEW

**Comment by rpsu** [comment]
With this PR merge I'll archive the repo, too.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # gh-project-actions
+
+**DEPRECATED**
+
+Github has very similar Action in their Marketplace, built by Github.
```
Comment (hugovk) [inline README.md:5]:
```suggestion
GitHub has a very similar Action in their Marketplace, built by GitHub. 
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
 # gh-project-actions
+
+**DEPRECATED**
+
+Github has very similar Action in their Marketplace, built by Github. 
+
+<https://github.com/marketplace/actions/add-to-github-projects>
+
+Functionally only difference is you can not specify the target column.
+However the project board can have a default column for new issues and PR's.
```
Comment (hugovk) [inline README.md:10]:
```suggestion
Functionally the only difference is you cannot specify the target column.
However the project board can have a default column for new issues and PRs.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+
+<https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#configure-built-in-automation>
+
+--
+
 Github actions for automatic management of Github project
```
Comment (hugovk) [inline README.md:16]:
```suggestion
GitHub actions for automatic management of GitHub project
```

**Review by hugovk — APPROVED** [review]
Minor wording suggestions.

**Comment by hugovk** [comment]
> With this PR merge I'll archive the repo, too.

Good idea.

We can also open issues in the repos that use this action, or ping them in another way, so they can update.

I won't list private repos in this public issue tracker, but you can see them here:

https://github.com/search?q=Yleisradio%2Fgh-project-actions&type=code

**Review by tjjalava — APPROVED** [review]

**Pull request merged**